### ISSUE TITLE
Varnam IME for Mac

### ISSUE THREAD

**Issue by subins2000** [varnamproject/govarnam#8]
This will be simpler than Windows. You'll need XCode.

- [ ] Fork the source code of [Lipika IME](https://github.com/ratreya/lipika-ime), open it in XCode, play with it, understand how it works
- [ ] Integrate GoVarnam into it
  - [ ] Build for Mac. The output will be a `.dynlib` or `.so`. This will be easy. Just do `make nix`. Maybe add Makefile rules specifically for Mac
  - [ ] GoVarnam functions can be called from Swift https://stackoverflow.com/q/34669958/1372424
- [ ] Automate Mac build via CI

**Comment by gotjoshua** [comment]
Hello! Do you have a working prototype for mac ime? 
build instructions?
Thanks!

**Comment by subins2000** [comment]
@gotjoshua Yes, you can install mac IME from here : https://varnamproject.com/download/mac/

**Comment by gotjoshua** [comment]
many thanks! 
i am actually investigating a way to configure a generic IME to use for training "chording"
so in addition to installing the actual varnam mac ime, i want to know how to create entries in the IME for english to english chording hints.
perhaps i ought to make a separate issue to ask for help with this, but i think this thread is still the correct place to ask for build instructions.

Can you tell me is the installable varnam ime for mac compiled from this govarnam repo?
how can i make changes to the dictionary and compile it for myself?
